coretex/entities/task_run/utils.py

```diff
@@ -58,4 +64,44 @@ def fetchDataset(datasetType: Type[Dataset], value: Any) -> Optional[Dataset]:
     if issubclass(datasetType, NetworkDataset):
         return datasetType.fetchById(value)
 
-    return None
\ No newline at end of file
+    return None
+
+
+def getSnapshotFiles(dirPath: Path, ignoredFiles: List[str]) -> List[Path]:
+    snapshotFiles: List[Path] = []
+    if (dirPath / ".coretexignore").exists():
+        return []
+
+    for path in dirPath.iterdir():
+        if path.is_dir():
+            snapshotFiles.extend(getSnapshotFiles(path, ignoredFiles))
+        elif str(path) not in ignoredFiles:
+            snapshotFiles.append(path)
+
+    return snapshotFiles
+
+
+def uploadSnapshot(entryPoint: Optional[str]) -> List[FileData]:
```

Comment:
Rename this function to "createSnapshot" and return snapshot path not `List[FileData]`